### PR TITLE
Remove queue limit from ACLK sync event loop

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
- Code cleanup
- Remove queue limit from ACLK sync event loop

### CODE REVIEW

**Review by thiagoftsm — APPROVED** [review]
PR is working as  expected, LGTM!

File: database/sqlite/sqlite_aclk_alert.c [hunk 1]
```diff
@@ -13,52 +13,78 @@
...
+    sqlite3_stmt *res = NULL;
+
     char sql[ACLK_SYNC_QUERY_SIZE];
-    snprintfz(sql, ACLK_SYNC_QUERY_SIZE-1, SQL_UPDATE_FILTERED_ALERT, uuid_str, ae->unique_id, unique_id);
-    sqlite3_exec_monitored(db_meta, sql, 0, 0, NULL);
-    ae->flags |= HEALTH_ENTRY_FLAG_ACLK_QUEUED;
+    snprintfz(sql, ACLK_SYNC_QUERY_SIZE-1, SQL_UPDATE_FILTERED_ALERT, uuid_str);
+    int rc = sqlite3_prepare_v2(db_meta, sql, -1, &res, 0);
+    if (rc != SQLITE_OK) {
+        error_report("Failed to prepare statement when trying to check for alert variables.");
```
Comment (MrZammler) [inline database/sqlite/sqlite_aclk_alert.c:28]: Really small, but the text here could be different.

**Review by MrZammler — APPROVED** [review]
Checked alerts functionality with cloud, everything appears ok!

File: database/sqlite/sqlite_aclk_alert.c [hunk 2]
```diff
@@ -521,25 +537,18 @@ void aclk_send_alarm_configuration(char *config_hash)
 
 int aclk_push_alert_config_event(char *node_id __maybe_unused, char *config_hash __maybe_unused)
 {
-    int rc = 0;
+    int rc;
```
Comment (MrZammler) [inline database/sqlite/sqlite_aclk_alert.c:540]: Not mine, but codacy complains here about no value. Likely if the ifdef below does not apply, we return a random value?

**Pull request merged**